packages/blocks/src/_common/components/utils.ts

```diff
@@ -21,10 +21,6 @@ export function getQuery(
   if (nativeRange.startContainer !== nativeRange.endContainer) {
     return null;
   }
-  const textNode = nativeRange.startContainer;
-  if (textNode.nodeType !== Node.TEXT_NODE) {
-    return null;
-  }
```

Comment:
When `compositionend` event is emitted, the node in the current native range is a `ELEMENT_NODE` not a `TEXT_NODE` in MacOS
```
<div contenteditable="true" class=" inline-editor " data-v-root="true">
```